js/researches/english/EnglishParticiple.js

```diff
@@ -30,33 +28,15 @@ var includesIndex = function( precedingWords, participleIndex ) {
 
 	var precedingWordsEndIndices = [];
 	forEach( precedingWords, function( precedingWord ) {
-		var precedingWordsEndIndex = precedingWord.index + precedingWord.match.length;
+		// + 1 because the end word boundary is not included in the match.
+		var precedingWordsEndIndex = precedingWord.index + precedingWord.match.length + 1;
```

Comment:
What if the word boundary is 2 spaces by accident?

Reply:
Extra spaces don’t affect word recognition. The regex will still match correctly.